lib/ecto.ex

```diff
@@ -31,7 +31,7 @@ defmodule Ecto do
   the accompanying sample application.
 
   After exploring the documentation and guides, consider checking out the
-  ["What's new in Ecto 2.1"](http://pages.plataformatec.com.br/ebook-whats-new-in-ecto-2-0)
+  ["What's new in Ecto 2.1"](https://pages.plataformatec.com.br/the-little-ecto-cookbook)
```

Comment:
Probably change:

```suggestion
  ["The Little Ecto CookBook"](https://pages.plataformatec.com.br/the-little-ecto-cookbook)
```

It looks like the ebook is already about Ecto 3.2. 🙂